lib/datasets/afinn.rb

```diff
@@ -0,0 +1,45 @@
+require "csv"
+require "zip"
+require_relative "dataset"
+#require_relative "zip_extractor"
+
+module Datasets
+  class Afinn < Dataset
+    Record = Struct.new(:word,
+                        :valence)
+
+    def initialize
+      super()
+      @metadata.id = "afinn"
+      @metadata.name = "AFINN"
+      @metadata.url = "http://www2.imm.dtu.dk/pubdb/pubs/6010-full.html"
+      @metadata.licenses = ["Open Database License (ODbL) v1.0"]
```

Comment:
Could you use SPDX ID?

See also: https://spdx.org/licenses/

```suggestion
      @metadata.licenses = ["ODbL-1.0"]
```